<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-GLJP20NJ" failures="0" tests="1" name="runners.Get_All_Users_API_Step_Runner" time="2.828" errors="1" timestamp="11 Aug 2020 02:43:37 GMT">
  <testcase classname="runners.Get_All_Users_API_Step_Runner" name="feature" time="2.828">
    <error message="java.lang.AssertionError: The following asserts failed:
	Verifying JSON Path value list expected [21] but found [false],
	Verifying JSON Path value list expected [127.0.0.1] but found [false]" type="cucumber.runtime.CucumberException">
      <![CDATA[cucumber.runtime.CucumberException: java.lang.AssertionError: The following asserts failed:
	Verifying JSON Path value list expected [21] but found [false],
	Verifying JSON Path value list expected [127.0.0.1] but found [false]
	at cucumber.api.testng.TestNGCucumberRunner.runCucumber(TestNGCucumberRunner.java:69)
	at cucumber.api.testng.AbstractTestNGCucumberTests.feature(AbstractTestNGCucumberTests.java:21)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.lang.AssertionError: The following asserts failed:
	Verifying JSON Path value list expected [21] but found [false],
	Verifying JSON Path value list expected [127.0.0.1] but found [false]
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
	at step.Get_All_Users_API_Step_Driver.gather_report(Get_All_Users_API_Step_Driver.java:124)
	at âœ½.And Test Get All Users API Gather all soft /hard asertion report(src/main/java/features/Get_All_Users_API.feature:8)
]]>
    </error>
  </testcase> <!-- feature -->
</testsuite> <!-- runners.Get_All_Users_API_Step_Runner -->
